photutils/psf/tests/test_models.py

```diff
@@ -0,0 +1,31 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import division
+
+import numpy as np
+from astropy.modeling.models import Gaussian2D
+
+from .. import FittableImageModel
+
+try:
+    import scipy
+    HAS_SCIPY = True
+except ImportError:
+    HAS_SCIPY = False
+
+
```

Comment:
@mcara You need to skip this test if `scipy` is not installed.  Add the line
`@pytest.mark.skipif('not HAS_SCIPY')` here.